Shelfie/Shelfie/UI/RequestBook/BTRequestBookViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  BTRequestBookViewController.m
+//  Shelfie
+//
+//  Created by Claudia Haddad on 8/2/18.
+//  Copyright © 2018 BookTrader. All rights reserved.
+//
+
+#import "BTRequestBookViewController.h"
+#import "BTBookAPIManager.h"
+
+
+@interface BTRequestBookViewController ()
+
+@property (strong, nonatomic) NSString *isbn;
+@property (strong, nonatomic) IBOutlet UITextField *isbnField;
+
+
+@end
+
+@implementation BTRequestBookViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+//- (void) textEntered {
```

Comment:
Is this function used for anything? If not delete it since it is commented

Reply:
will be used in future!